datahub-web-react/src/App.tsx

```diff
@@ -43,6 +47,14 @@ const client = new ApolloClient({
 });
 
 const App: React.VFC = () => {
+    const [dynamicThemeConfig, setDynamicThemeConfig] = useState<Theme | null>(null);
```

Comment:
Why not just set this to default?

Reply:
I see - is the idea that you can specify a "partial" theme that just overrides the default? This makes sense